src/python/dxpy/templating/templates/nextflow/nextflow.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+on_exit() {
+  ret=$?
+  # upload log file
+  dx upload $LOG_NAME --path $DX_LOG --wait --brief --no-progress --parents || true
+  # backup cache
+  echo "=== Execution complete — uploading Nextflow cache metadata files"
+  dx rm -r "$DX_PROJECT_CONTEXT_ID:/.nextflow/cache/$NXF_UUID/*" 2>&1 >/dev/null || true
+  dx upload "/.nextflow/cache/$NXF_UUID" --path "$DX_PROJECT_CONTEXT_ID:/.nextflow/cache/$NXF_UUID" --no-progress --brief --wait -p -r || true
+  # done
+  exit $ret
+}
+
+  dx_path() {
+  local str=${1#"dx://"}
+  local tmp=$(mktemp -t nf-XXXXXXXXXX)
+  case $str in
+    project-*)
+      dx download $str -o $tmp --no-progress --recursive -f
+      echo file://$tmp
+      ;;
+    container-*)
+      dx download $str -o $tmp --no-progress --recursive -f
+      echo file://$tmp
+      ;;
+    *)
+      echo "Invalid $2 path: $1"
+      return 1
+      ;;
+  esac
+}
+    
+main() {
+    set -f
+
+    [[ $debug ]] && set -x && env | sort
+    [[ $debug ]] && export NXF_DEBUG=2
+    
+    if [ -n "$docker_creds" ]; then
+        dx download "$docker_creds" -o /home/dnanexus/credentials
+        ls /home/dnanexus
+        source /.dx.nextflow/resources/usr/local/bin/dx-registry-login
+    fi
+
+    LOG_NAME="nextflow-$(date +"%y%m%d-%H%M%S").log"
+    DX_WORK=${work_dir:-$DX_WORKSPACE_ID:/scratch/}
+    DX_LOG=${log_file:-$DX_PROJECT_CONTEXT_ID:$LOG_NAME}
+    export NXF_WORK=dx://$DX_WORK
+    export NXF_HOME=/opt/nextflow
+    export NXF_UUID=${resume_session:-$(uuidgen)}
+    export NXF_IGNORE_RESUME_HISTORY=true
+    export NXF_ANSI_LOG=false
+    export NXF_EXECUTOR=dnanexus
+    export NXF_PLUGINS_DEFAULT=nextaur@1.0.0
+    export NXF_DOCKER_LEGACY=true
+    #export NXF_DOCKER_CREDS_FILE=$docker_creds_file
+    #[[ $scm_file ]] && export NXF_SCM_FILE=$(dx_path $scm_file 'Nextflow CSM file')
+    trap on_exit EXIT
+    echo "============================================================="
+    echo "=== NF work-dir : ${DX_WORK}"
+    echo "=== NF Resume ID: ${NXF_UUID}"
+    echo "=== NF log file : ${DX_LOG}"
+    echo "=== NF cache    : $DX_PROJECT_CONTEXT_ID:/.nextflow/cache/$NXF_UUID"
+    echo "============================================================="
+
+    filtered_inputs=""
+    
+    @@RUN_INPUTS@@
+    nextflow -trace nextflow.plugin $nf_run_args_and_pipeline_params -log ${LOG_NAME} run /home/dnanexus/nfp @@PROFILE_ARG@@ -name ${NXF_UUID} $nf_run_args_and_pipeline_params ${filtered_inputs}
```

Comment:
```suggestion
    nextflow -trace nextflow.plugin $nf_run_args_and_pipeline_params -log ${LOG_NAME} run /home/dnanexus/nfp @@PROFILE_ARG@@ -name $test-{NXF_UUID} $nf_run_args_and_pipeline_params ${filtered_inputs}
```
https://staging.dnanexus.com/projects/GG1P1p80728ypkgp9vyYg2kX/monitor/job/GG2byzQ0728QkYvY78bGfBjB
ran into error saying 

> Not a valid run name: `76f125c8-637e-4696-a51f-ddcf8d72b8ce` -- It must match the pattern ^[a-z](?:[a-z\d]|[-_](?=[a-z\d])){0,79}$